context/movieData.context.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
```

Comment:
todo: When creating a React Context, the file should be Capitalized too and match the component name. In this case, should be `MoviesContext.tsx`